show/interfaces/__init__.py

```diff
@@ -537,33 +537,30 @@ def errors(ctx, interfacename):
 
     port_operr_table = get_all_port_errors(interfacename)
 
-    # Define a list of all potential errors
+    # Define a list of all potential errors's DB entries
     ALL_PORT_ERRORS = [
-        "oper_error_status",
-        "mac_local_fault",
-        "mac_remote_fault",
-        "fec_sync_loss",
-        "fec_alignment_loss",
-        "high_ser_error",
-        "high_ber_error",
-        "data_unit_crc_error",
-        "data_unit_misalignment_error",
-        "signal_local_error",
-        "crc_rate",
-        "data_unit_size",
-        "code_group_error",
-        "no_rx_reachability"
+        ("oper_error_status", "oper_error_status_time"),
```

Comment:
@bobby-nexthop you want each error to indicate the time of occurrence? Can you change the title of the PR accordingly?